mlflow/store/tracking/dbmodels/models.py

```diff
@@ -150,6 +150,10 @@ class SqlRun(Base):
         PrimaryKeyConstraint('run_uuid', name='run_pk')
     )
 
+    @staticmethod
+    def get_attribute_name(mlflow_attribute_name):
```

Comment:
Can we adds docs to this function as follows?:
```
    @staticmethod
    def get_attribute_name(search_attribute_name):
        """
        Resolves an MLflow Search attribute name to a `SqlRun` attribute name.
        """
        # Currently, MLflow Search attributes defined in `SearchUtils.VALID_SEARCH_ATTRIBUTE_KEYS`
        # share the same names as their corresponding `SqlRun` attributes. Therefore, this function
        # returns the same attribute name
        return search_attribute_name
```

Reply:
Done